src/patients/allergies/Allergies.tsx

```diff
@@ -46,18 +49,30 @@ const Allergies = (props: AllergiesProps) => {
         </div>
       </div>
       <br />
+      <Switch>
+        <Route exact path="/patients/:id/allergies">
+          <List>
+            {patient.allergies?.map((a: Allergy) => (
+              <ListItem
```

Comment:
We should indicate that the list item is clickable. We can do this by adding `action` as a prop to `ListItem`